api/python/quilt3/data_transfer.py

```diff
@@ -512,6 +506,10 @@ def list_objects(bucket, prefix, recursive=True):
         return prefixes, objects
 
 
+def _looks_like_dir(s):
+    return not s or s.endswith('/')
```

Comment:
This won't work correctly on Windows, right?

Reply:
It's only used on URLs, so it's going to be `/`